sdk/core/abort-controller/package.json

```diff
@@ -19,8 +19,8 @@
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint \"src/**/*.ts\" \"test/**/*.ts\" -c ../../.eslintrc.old.json --fix --fix-type [problem,suggestion]",
-    "lint": "eslint -c ../../.eslintrc.old.json src test --ext .ts -f html -o abort-controller-lintReport.html || exit 0",
+    "lint:fix": "eslint  -c ../../../.eslintrc.json package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
```

Comment:
same comment about `/.eslintrc.json` as I had in the core-lro PR: https://github.com/Azure/azure-sdk-for-js/pull/11273#discussion_r489490221